cmd/subrouter/sr.go

```
@@ -500,10 +588,49 @@ func (r srRunner) addProvider(ctx context.Context, args []string) error {
+	return agentgrok.DefaultStore()
+}
+
+// grokSignIn keeps Subrouter's routed subscription credential independent from
+// the Grok CLI's single global auth file. Human authorization happens before
+// the short account-disk transaction that makes the new credential visible to
+// running workers.
+func (r srRunner) grokSignIn(ctx context.Context) error {
+	store := r.grokStore()
+	credential, err := store.Authorize(ctx, r.client, r.out)
+	if err != nil {
+		return fmt.Errorf("Grok login failed: %w", err)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)